docs/language/resources.mdx

```diff
@@ -12,18 +12,13 @@ Resources **must** be created (instantiated) by using the `create` keyword.
 At the end of a function which has resources (variables, constants, parameters) in scope,
 the resources **must** be either **moved** or **destroyed**.
 
-They are **moved** when used as an initial value for a constant or variable,
-when assigned to a different variable,
-when passed as an argument to a function,
-and when returned from a function.
+They are **moved** when used as an initial value for a constant or variable, when assigned to a different variable, when passed as an argument to a function, and when returned from a function.
```

Comment:
Here and below: It looks like only the newlines/line breaks got removed from the original content.

The line breaks are intentional, they are "Semantic Line Breaks": They are ignored in the rendered output, but are useful in the source Markdown, because they keep lines limited to a certain length (<=100) like in source code, and make it easier to review changes.

See https://sembr.org/

Reply:
I've worked with both methods before and find removing semantic line breaks to generally improve workflow for writing, editing, and updating docs.

Most editors can be set to wrap the text when viewing it, and by not having the line breaks it's much easier to edit and wordsmith the text because updates automatically display similar to how they'd appear on the page without having to save to autoformat, or run a script to rebreak at 80 wide.